pipeline/pipelines/gapic_generation_pipeline.py

```diff
@@ -52,20 +51,26 @@ def __init__(self, **kwargs):
             **kwargs)
 
 
+class GapicClientBatchPipeline(batch_gen.BatchPipeline):
+
+    def get_pipeline_tasks(self, **kwargs):
+        task_factory = get_gapic_task_factory(kwargs['language'])
+        return task_factory._get_gapic_codegen_tasks(**kwargs)
```

Comment:
Let's talk about this this afternoon. I have already begun working on a new command-line interface for artman, and this work is running up against that.